packages/jumpstarter/jumpstarter/streams/progress.py

```
@@ -1,131 +1,76 @@
-                self.__send = tqdm(desc="transfer", **TQDM_KWARGS)
-
-        self.__send.update(len(item))
+            self.__prog.start()
+            self.__send = self.__prog.add_task("transfer", total=None)
+
+        self.__prog.advance(self.__recv, len(item))
+        if self.logging and (datetime.now() - self.__last > timedelta(seconds=2)):
+            self.__last = datetime.now()
+            console = Console(file=StringIO())
+            console.print(self.__prog.get_renderable())
+            logger.info(console.file.getvalue())
```

Pylint (3.3.7)

[error] Instance of 'TextIOWrapper' has no 'getvalue' member

(E1101)